src/software/ai/navigator/navigator.cpp

```diff
@@ -352,3 +322,13 @@ std::vector<std::vector<Point>> Navigator::getPlannedPaths()
 {
     return planned_paths;
 }
+
+void Navigator::addPlannedPaths(const Path &path)
+{
+    std::vector<Point> points;
+    for (double i = 0.0; i <= path.size(); i++)
```

Comment:
Up above I thought `1.0` was the final position on the path, but down here it looks like it's just some sort of index? Bit confused.

Reply:
Spline takes in points and assigns each point to a unit of time (e.g. time=0 -> point 1, time=1 -> point 2, ...) and interpolates with line segments-- this isn't correct in terms of robot speed, but it is easiest to integrate into current navigator. It's in the description... should this be a comment somewhere?

Reply:
Splines should have no knowledge of time. When you think what a `spline` is, it's just a mapping from some subset of `R^1` to a subset of `R^2`.

Reply:
done

Reply:
just be clear, splines are from 0 to 1 now right?

Reply:
yes